source/matrix_free/evaluation_template_factory.inst.in

```diff
@@ -0,0 +1,82 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 2020 by the deal.II authors
+//
+// This file is part of the deal.II library.
+//
+// The deal.II library is free software; you can use it, redistribute
+// it, and/or modify it under the terms of the GNU Lesser General
+// Public License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+// The full text of the license can be found in the file LICENSE.md at
+// the top level directory of deal.II.
+//
+// ---------------------------------------------------------------------
+
+
+for (deal_II_dimension : DIMENSIONS;
+     deal_II_scalar_vectorized : REAL_SCALARS_VECTORIZED)
```

Comment:
This is the reason why it gets expensive - we need to compile it for all `VectorizedArray` types we have. We could again think of a shortcut to only use fast paths for some degrees, but I fear that we would need to be careful with describing this for the user to not give surprises when performance differs by a factor of 2.